cluster/gce/windows/configure.ps1

```diff
@@ -139,6 +139,7 @@ try {
 
   Set-PrerequisiteOptions
   $kube_env = Fetch-KubeEnv
+  Configure-WindowsDefender
```

Comment:
Invoking this cmdlet in `Set-PrerequisiteOptions` seems to be more appropriate. I can see this function expanding to include other things like ignore directories etc.

Reply:
ok. I'll move it